src/omero_cli_render.py

```diff
@@ -98,8 +98,8 @@
         label: <string>     Channel name
         start: <float>      Start of rendering window, optional (needs end)
         end: <float>        End of rendering window, optional (needs start)
-        min: <float>        Minimum pixel value intensity, optional
-        max: <float>        Maximum pixel value intensity, optional
+        min: <float>        Min pixel intensity, optional (needs max if unset)
+        max: <float>        Max pixel intensity, optional (needs min if unset)
```

Comment:
"if set" ?

Reply:
I wanted to say `(need to set max if max is unset)` but line length too long

Reply:
What about something like

```
        min: <float>        Min pixel intensity, required if max is set
        max: <float>        Max pixel intensity, required if min is set
```

to communicate the conditional requirement?

Reply:
What I'm trying to say here is that if the channel has no `StatsInfo` then you need to set `min` AND `max`. This is the same as setting start/end, where we use the format:
`start: <float>      Start of rendering window, optional (needs end)`

But if the channel already has `StatsInfo` the you can set `min` or `max` independently.

I don't know if it's important to document this 2nd case. I don't think it's possible to explain that in the one-line we have here, so maybe we should just be consistent and do:
`max: <float> Maximum pixel intensity, optional (needs min)`

Reply:
Understood and I certainly missed the subtle conditional behavior.

I think it's too much to try and describe in such a short field description, maybe we just mention `, optional` and have a dedicated text paragraph below to expand on the two  use cases i.e. whether stats info is null or not. Is there a way to know whether stats info is null?